### PR TITLE
Added  Automatically Updated contributors section using github action and also added CODE_OF_CONDUCT

### PR DESCRIPTION
**Pull request by sb-decoder**
@makoteq please check this PR and if you want this changes in your repo then merge this. Thank You.

### CODE REVIEW

**Comment by makoteq** [comment]
Hi! @sb-decoder 
Interesting idea,
Could you make a new section for contributors and put "contrib rocks" img in it?
I think that would make it look better
After that im willing to merge this one

**Comment by sb-decoder** [comment]
> Hi! @sb-decoder Interesting idea, Could you make a new section for contributors and put "contrib rocks" img in it? I think that would make it look better After that im willing to merge this one

@makoteq  I have made a PR , please check if your query solved or not.

**Comment by sb-decoder** [comment]
@makoteq Please check this one.

File: README.md [hunk 1]
```diff
@@ -34,10 +34,8 @@ metaxa read <text> [<delay>]
 ## :raised_hands: Contributing 
 Contributions are welcome, check [Issues!](https://github.com/makoteq/metaxa/issues)</br></br>
 **If you like the project make sure to hit :star:**
-## Thanks to these amazing people
-<a href="https://github.com/makoteq/metaxa/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=makoteq/metaxa&&max=817" />
-</a>
+## ✨ Contributors
```
Comment (makoteq) [inline README.md:37]: Fix typo, link doesnt work

**Review by makoteq — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sb-decoder) [reply README.md:37]: After merge , the link will work finely

**Comment by sb-decoder** [comment]
@makoteq resolved issue

File: README.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ metaxa read <text> [<delay>]
 Contributions are welcome, check [Issues!](https://github.com/makoteq/metaxa/issues)</br></br>
 **If you like the project make sure to hit :star:**
 ## ✨ Contributors 
-Chcekout our [Contributors](https://github.com/makoteq/metaxa/blob/main/CONTRIBUTORS.md)
+Chcekout our [Contributors](https://github.com/makoteq/metaxa/blob/master/CONTRIBUTORS.md)
```
Comment (makoteq) [inline README.md:38]: fix `Chcekout ` typo

**Review by makoteq — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (makoteq) [reply README.md:38]: and link still doesnt work!

**Comment by makoteq** [comment]
Please test your changes before submitting

**Comment by sb-decoder** [comment]
> Please test your changes before submitting

I fixed typo mistake and link will start working after you merge it.

**Comment by sb-decoder** [comment]
Link is not working now because you still have not the CONTRIBUTORS.md in your repo. So it can't find it.

**Comment by makoteq** [comment]
good work!

**Pull request merged**